components/Error.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from "../styles/components/Error.module.css";
+import React from "react";
+
+// Error text
```

Comment:
Can we change the comment to something like "Error displayed when calls to the backend fail"